manifests/4.4/cluster-logging.v4.4.0.clusterserviceversion.yaml

```diff
@@ -299,15 +299,15 @@ spec:
                   - name: OPERATOR_NAME
                     value: "cluster-logging-operator"
                   - name: ELASTICSEARCH_IMAGE
-                    value: "quay.io/openshift/origin-logging-elasticsearch5:latest"
+                    value: registry.svc.ci.openshift.org/ocp/4.4:logging-elasticsearch5
```

Comment:
should we be changing these to use the ci registry?

Reply:
Please revert this change.  @ewolinetz I think maybe there is something in our hack stricts that is making this mod to this file instead of some other

Reply:
reverted this file